### DISCUSSION TITLE
Tips for getting good TTS output

### DISCUSSION THREAD

**Discussion by RuABraun** [coqui-ai/TTS#925] [General Q&A] [answered]
Hello, I'm trying out the DE TTS model.

So far the output has been disappointing in quality, and for some reason the wavfile is always around 23.5s in length.

I don't care about RTF. Are there some options I can set to go for maximum quality?

Seems from the model list that the focus has been on English. If I were to train a model for German what model type would you recommend, considering the fact that I only care about quality and not speed?

E: Okay I just realized I'm supposed to put a "." at the end :smiley: 

E2: Is there a way I could pass in a list of phones?

**Comment by thorstenMueller** [comment, marked as answer]
Hi @RuABraun .
Even if there's plenty of room for quality improvements i hope your "disapointment" is less after you added a dot at the end of the phrase ;-).
As far i know it's not supported to give phonemes as text input (if that's meant by you).

**Reply by RuABraun** [reply]
Thank you for the response! :)

Any recommendation on what's the best model type (to train for myself) if I only care about quality and not speed?

**Reply by thorstenMueller** [reply]
When i'm right Taco2 DDC and Hifigan is a good combination.

**Reply by RuABraun** [reply]
Thank you!

**Reply by RuABraun** [reply]
Which models are multispeaker? I tried a bunch out but always got an error implying there were no different speaker IDs available when calling `tts --model_name <name>  --list_speaker_idxs`

Also, the models are trained with phones as input right (I noticed a bunch of dependencies for getting phones from graphemes)? So it should be possible for me to make some code changes so I can decide what phones go in right? Or do some models work with letters?
